documentation/docs/lab-share-lib.md

```diff
@@ -0,0 +1,116 @@
+# lab-share-lib
+
+`lab-share-lib` is a separate [library](https://github.com/sanger/lab-share-lib) that encapsulates AMQP connectivity logic, message deserialization and schema validation. Sections below are some discussions on how it encapsulates logic related to messaging infrastructure and schema validation.
```

Comment:
I noticed there are some cases where you've put in a new line after the end of a sentence, and some where you haven't.  Not sure if you want to take the time to tweak these.